### ISSUE TITLE
Cannot duplicate "entity" part

### ISSUE THREAD

**Issue by friggityfroggy** [CapsAdmin/pac3#1161]
When setting a group to an owner with multiple instances, such as "frag" for thrown grenade skins, any "entity" part that comes out will only apply to the most recent entity rather than freely duplicating like it does with added models.

**Comment by thegrb93** [comment]
Example outfit? I'm not sure how to do what you're describing

**Comment by CapsAdmin** [comment]
If I understand this, I think this is expected behavior. You need to check duplicate on the group for it to work.

**Comment by thegrb93** [comment]
Does group have this capability? I didn't really see anything in its source code that does this.

**Comment by thegrb93** [comment]
https://github.com/CapsAdmin/pac3/blob/master/lua/pac3/core/client/parts/group.lua#L73-L95

**Comment by CapsAdmin** [comment]
Yes something is definetly wrong here, I'll have a look at it soon.

I think while I'm at it, I'll make it so duplicate applies on all entities that match, right now it only applies to the ones that spawn after you check duplicate.
